build.gradle

```diff
@@ -46,6 +47,29 @@ ext.compatibilityVersion = '1.7'
 sourceCompatibility = compatibilityVersion
 targetCompatibility = compatibilityVersion
 
+sourceSets {
+    main.java.srcDirs = ['src/main/java'
+                        ,'src/generated/java' ]
```

Comment:
I assume `src/generated/java` is an eclipse thing? Is the need for this to pickup the `AutoValue*` classes?